core-galleon-pack/src/main/resources/content/README.txt

```diff
@@ -0,0 +1,83 @@
+ _    _  _  _      _ ______  _           __   _____
+| |  | |(_)| |    | ||  ___|| |         /  | |  _  |
+| |  | | _ | |  __| || |_   | | _   _   `| | | |/' |
+| |/\| || || | / _` ||  _|  | || | | |   | | |  /| |
+\  /\  /| || || (_| || |    | || |_| |  _| |_\ |_/ /
+ \/  \/ |_||_| \__,_|\_|    |_| \__, |  \___/ \___/
```

Comment:
Perhaps we should remove the 10 part here

Reply:
If we do we must remove it in the same file in core-feature-pack as well, as a goal of this PR is to produce a build where the galleon build output is the same as the legacy build.

If Alexey wants to do that, ok, but in general I don't think we should make fixing existing problems a requirement of this PR.